stats/tinymembench

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
```

Comment:
help add rspec test under spec/stats

Reply:
Added rspec test under spec/stats for tinymembench.